js/client-battle-tooltips.js

```diff
@@ -213,7 +213,7 @@ var BattleTooltips = (function () {
 				text += '' + (move.desc || move.shortDesc) + '</p>';
 
 				if ('defrost' in move.flags) {
-					text += '<p>The user thaws out if it is frozen.</p>';
+					text += '<p class="movetag">The user thaws out if it is frozen.</p>';
```

Comment:
Actually, because it's copy/pasted from PSdex. We can leave it in for Stylish users.